misk-hibernate/src/main/kotlin/misk/vitess/StartVitessService.kt

```diff
@@ -255,13 +256,24 @@ class DockerVitessCluster(
     }
   }
 
+  /**
+   * Set binary logging to row based.
+   */
+  private fun setRowBasedBinaryLogging() {
+    cluster.openMysqlConnection().use { connection ->
+      connection.createStatement().use { statement ->
+        statement.execute("SET GLOBAL binlog_format = 'ROW'")
+      }
+    }
+  }
+
   fun stop() {
     if (!isRunning) {
       return
     }
     isRunning = false
 
-    docker.removeContainerCmd(containerId!!).withForce(true).exec()
+    docker.removeContainerCmd(containerId!!).withForce(true).withRemoveVolumes(true).exec()
```

Comment:
we should port vitess to use the container `Composer` in `misk-testing` and add this flag so other containers don't make the same mistakes (like zk and kafka)